src/pages/Chart/ChartBox.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function ChartBox(props) {
+  const [movieChart, setMovieChart] = props;
```

Comment:
구조분해 할당이 잘못 된것 같습니다!
```suggestion
  const {movieChart, setMovieChart} = props;
```

Reply:
넵

Reply:
수정 사항 반영했는데 따로 PR올려야하는 것으로 뜹니다 올려도 되나요?